exammode/templates/exammode/staff/exam_management.html

```diff
@@ -0,0 +1,90 @@
+{% extends "course/course_base.html" %}
+{% load i18n %}
+{% load course %}
+{% load static %}
+
+{% block title %}{% trans "Exam management" %} | {{ block.super }}{% endblock %}
+{% block view_tag %}Exam management{% endblock %}
+
+{% block breadcrumblist %}
+{{ block.super }}
+<li class="active">{% trans "Exam management" %}</li>
+{% endblock %}
+
+{% block columns %}
+
+<h3>Add new exam session</h3>
+
+<form method="post">
+	{% csrf_token %}
+	{{ form.non_field_errors }}
+	<div class="contentDividerExamSession">Content</div>
+	<div class="fieldWrapperExamSession">
+		{{ form.course_instance.errors }}
+		<label for="{{ form.course_instance.id_for_label }}">Course Instance:</label>
```

Comment:
Instead of typing HTML manually, forms labels can be rendered with `form.course_instance.label_tag`.
https://docs.djangoproject.com/en/2.2/topics/forms/#rendering-fields-manually
https://docs.djangoproject.com/en/2.2/ref/forms/api/#django.forms.BoundField.label_tag

Reply:
I guess `label_tag` would also translate the name if the field has a translated verbose name.

Reply:
https://docs.djangoproject.com/en/2.2/ref/models/fields/#verbose-name

Reply:
This seems to be fixed.